third_party/2/concurrent/futures/__init__.pyi

```diff
@@ -15,27 +15,27 @@ class Future(Generic[_T]):
     def running(self) -> bool: ...
     def done(self) -> bool: ...
     def result(self, timeout: Optional[float] = ...) -> _T: ...
-    def exception(self, timeout: Optional[float] = ...) -> Any: ...
+    def exception(self, timeout: Optional[float] = ...) -> Optional[BaseException]: ...
```

Comment:
Actually the Any here is intentional; it was added to support old-style classes (#946).